benchopt/runner.py

```diff
@@ -336,6 +336,8 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
         by the objective is not the same for all parameters, the missing data
         is set to `NaN`.
     """
+    print("BenchOpt is running", flush=True)
```

Comment:
Not sure why you use `flush` here? I think the flush is autmatic when ending the line so I would simply remove if it is not needed.
```suggestion
    print("BenchOpt is running")
```

Reply:
Fixed in d65f300

It was a copy/paste actually. Since flush was used elsewhere, I assumed it was important.